sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/AppConfigurationFeatureManagementPropertySource.java

```diff
@@ -127,7 +130,17 @@ List<ConfigurationSetting> getFeatureFlagSettings() {
     @SuppressWarnings("unchecked")
     private Object createFeature(FeatureFlagConfigurationSetting item) {
         String key = getFeatureSimpleName(item);
-        Feature feature = new Feature(key, item);
+        String requirementType = "Any";
+        try {
+            JsonNode node = CASE_INSENSITIVE_MAPPER.readTree(item.getValue());
+            JsonNode conditions = node.get("conditions");
+            if (conditions != null && conditions.get("requirement_type") != null) {
```

Comment:
nit: Should we consider define this property name `requirement_type` as a constant as well?

Reply:
Why it's underscore in requirement_type, but in the pojo it's requirement-type?